src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -249,6 +249,36 @@ s! {
         pub ch_size: ::Elf32_Word,
         pub ch_addralign: ::Elf32_Word,
     }
+
+    pub struct timex {
+        pub modes: ::c_uint,
+        pub offset: ::c_long,
+        pub freq: ::c_long,
+        pub maxerror: ::c_long,
+        pub esterror: ::c_long,
+        pub status: ::c_int,
+        pub constant: ::c_long,
+        pub precision: ::c_long,
+        pub tolerance: ::c_long,
+        pub time: ::timeval,
+        pub tick: ::c_long,
+        pub ppsfreq: ::c_long,
+        pub jitter: ::c_long,
+        pub shift: ::c_int,
+        pub stabil: ::c_long,
+        pub jitcnt: ::c_long,
+        pub calcnt: ::c_long,
+        pub errcnt: ::c_long,
+        pub stbcnt: ::c_long,
+        pub tai: ::c_int,
+        pub __padding: [::c_int; 11],
```

Comment:
A padding field doesn't need to be public generally, is there any reason?

Reply:
I suppose it doesn't matter much, but if this one should be private then I should've made the [glibc](https://docs.rs/libc/0.2.99/libc/struct.timex.html) ones private as well when I contributed #1642. So I can either keep this one public, be inconsistent, or make a breaking change to the glibc version. Up to you, I have no strong feelings on the matter.

Reply:
Ah makes sense! Let's keep it as-is for now.